tests/Integration/Behaviour/Features/Context/Domain/CustomerServiceFeatureContext.php

```diff
@@ -208,15 +217,138 @@ public function deleteThread(string $threadReference): void
     public function assertThreadIsDeleted(string $threadReference): void
     {
         /** @var CustomerThread $customerThread */
-        $customerThread = SharedStorage::getStorage()->get($threadReference);
+        $customerThread = $this->getSharedStorage()->get($threadReference);
 
         try {
             $query = new GetCustomerThreadForViewing((int) $customerThread->id);
             $this->getQueryBus()->handle($query);
 
             throw new NoExceptionAlthoughExpectedException(sprintf('Thread %s exists, but it was expected to be deleted', $threadReference));
         } catch (CustomerThreadNotFoundException $e) {
-            SharedStorage::getStorage()->clear($threadReference);
+            $this->getSharedStorage()->clear($threadReference);
+        }
+    }
+
+    /**
+     * @Then contact :contactReference should have :expectedThreads threads
+     *
+     * @param string $contactReference
+     * @param int $expectedThreads
+     */
+    public function assertContactHasThreads(string $contactReference, int $expectedThreads): void
+    {
+        $contactId = $this->referenceToId($contactReference);
+        $customerServiceSummaries = $this->getQueryBus()->handle(
+            new GetCustomerServiceSummary()
+        );
+
+        /** @var CustomerServiceSummary $customerServiceSummary */
+        foreach ($customerServiceSummaries['summaries'] as $customerServiceSummary) {
```

Comment:
The returned data is an object now, so you need to adapt all its usages including in behat tests, that's probably why they are failing right now